hack/build/bazel-docker.sh

```diff
@@ -109,6 +109,11 @@ else
     volumes="-v ${BUILDER_VOLUME}:/root:rw,z,exec"
 fi
 
+if [ ${CI} == "true" ]; then
```

Comment:
do we ever set this though?

Reply:
Its an environment variable that gets added by prow.
```
						{
							"name": "CI",
							"value": "true"
						},
```
([source](https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/2474/pull-cdi-unit-test/1595070406727831552/podinfo.json))

Reply:
Prow doc - https://github.com/kubernetes-sigs/prow/blob/main/site/content/en/docs/Legacy%20Snapshot/prow/jobs.md#job-environment-variables